templates/annotation_guidelines_button.html

```diff
@@ -5,21 +5,11 @@
             aria-controls="collapse-guidelines">Hide Instructions</button>
     </p>
     <div class="collapse in" id="collapse-guidelines" aria-expanded="true">
-    <div class="card card-body guidelines" id="annotations-guidelines-content">
+    <div class="guidelines" id="annotations-guidelines-content">
     </div>
 </div>
 {% endif %}
 
-<script type="text/javascript">
-  $(".collapse-guidelines-button").on('click', function(){
-      var showInstructionsMsg = 'Show Instructions';
-      var hideInstructionsMsg = 'Hide Instructions';
-      $(this).text(function(i, v){
-          return v === showInstructionsMsg ? hideInstructionsMsg : showInstructionsMsg
```

Comment:
Since the variables are only being used a single time, it might be more readable to leave them out and just include the plain text here. It may also help to clean-up the ternary just a tiny bit as shown below. :) Thank you.

```js
$(this).text((i, v) => {
  return `${v.startsWith('Show') ? 'Hide' : 'Show'} Instructions`;
});
```